src/Daily-Hydration.js

```diff
@@ -0,0 +1,15 @@
+class DailyHydration {
```

Comment:
We Decided that it would not be the best idea to start with the DailyHydration, as explained in our thoughts about data flow above.